### PR TITLE
Removed deprecated images from the list

### PR DESCRIPTION
**Pull request by LaurentGoderre**

### CODE REVIEW

File: doi.jq [hunk 1]
```diff
@@ -45,6 +41,12 @@ def repos_anti_subset:
 		"rakudo-star",
 		"silverpeas",
 
+		# deprecated
+		"adminer",
+		"centos", # direct children: eclipse-temurin ibm-semeru-runtimes percona swift
+		"php-zendserver",
+		"sl",
```
Comment (whalelines) [inline doi.jq:48]: The sl bashbrew file has already been removed from docker-library/official-images/library.

**Review by whalelines — COMMENTED** [review]
Rather than adding the other three here, should we just remove their bashbrew files from docker-library/official-images?

File: doi.jq (hunk 1 above)
Comment (LaurentGoderre) [reply doi.jq:48]: Im more grouping them at the bottom until they can safely be removed

**Comment by LaurentGoderre** [comment]
Im more grouping them at the bottom until they can be safely removed.

**Comment by tianon** [comment]
Agree - we can instead do the two-step and remove them: https://github.com/docker-library/official-images/pull/17276 :eyes:

**Review by tianon — APPROVED** [review]

**Pull request merged**